Resources/ConfigPresets/Impstation/impstation.toml

```diff
@@ -51,3 +51,9 @@ role_time = 10
 [tips]
 # this doesn't actually propagate to the client right now
 login_dataset = "LoginTips"
+
+[whitelist]
+enabled = true
+# pretty much only worries about users who've been manually whitelisted
+# all players who have joined once before ~4am UTC 19th Apr 2025 should be whitelisted at time of edit
+prototype_list = "basicWhitelist"
```

Comment:
all fine but you can omit prototype_list cause this is already the default

https://github.com/impstation/imp-station-14/blob/master/Content.Shared/CCVar/CCVars.Whitelist.cs#L18

I was away on the weekend